app/helpers/drug_stock_helper.rb

```diff
@@ -6,4 +6,31 @@ def drug_stock_region_label(region)
       region.localized_region_type.capitalize
     end
   end
+
+  def filter_params
+    params[:zone].present? || params[:size].present?
+  end
+
+  def accessible_organization_facilities
+    if CountryConfig.current_country?("Bangladesh")
+      Organization.joins(facility_groups: :facilities).where(facilities: {id: @accessible_facilities}).distinct.pluck(:slug).include?("nhf")
+    else
+      true
+    end
```

Comment:
Could we remove country-specific logic from the code? Let's move this into some config

Reply:
This applies to the rest of the file as well